project/util/lob_models_util.py

```diff
@@ -81,3 +81,8 @@ class Meta:
         blank=True,
         help_text="When the letter was fully processed, i.e. sent to all relevant parties.",
     )
+
+    pdf_base64 = models.TextField(
```

Comment:
Is there a particular reason this has to go on `LocalizedHTMLLetter` and not on the LALetterBuilder-specific object called `Letter` in `laletterbuilder/models.py`? It seems like we are not surfacing those PDFs on the NoRent frontend, so is there some other utility of having them saved?